packages/typespec-azure-portal-core/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "@azure-tools/typespec-azure-portal-core",
+  "version": "1.0.0",
+  "description": "",
+  "scripts": {
+    "build": "tsc -p .",
+    "test": "vitest run",
+    "test:watch": "vitest -w",
+    "test-official": "vitest run --coverage --reporter=junit --reporter=default --no-file-parallelism"
+  },
+  "keywords": [
+    "Azure",
+    "Portal",
+    "TypeSpec"
+  ],
+  "author": "",
+  "license": "MIT",
```

Comment:
```suggestion
  "license": "MIT",
  "homepage": "https://azure.github.io/typespec-azure",
  "readme": "https://github.com/Azure/typespec-azure/blob/main/packages/typespec-azure-portal-core/README.md",
```

Reply:
ah! gotcha!! okay!! will change the package.json now :)